src/main/java/ua/khshanovskyi/feign/NasaFeign.java

```diff
@@ -0,0 +1,16 @@
+package ua.khshanovskyi.feign;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import ua.khshanovskyi.entity.Photos;
+
+@FeignClient(name = "nasa-client", url = "https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos")
```

Comment:
I think it's better to get feign properties(name, url) from application properties, so when the name or URL of service changes it only needs to update the properties file.